Source/WebCore/rendering/RenderElement.h

```diff
@@ -311,6 +311,7 @@ class RenderElement : public RenderObject {
 
     virtual bool establishesIndependentFormattingContext() const;
     bool createsNewFormattingContext() const;
+    bool isBlockLevelElementWithBlockStepSizing() const;
```

Comment:
do we really need this API for this very specific case?

Reply:
Not necessarily. It was mostly helpful when reading through all of the conditions in establishesIndependentFormattingContext. Tim's suggestion would be just as good or it could even be an lambda